docs/how-to-guides/new-user-guides/kubernetes-resources-setup/load-balancer-and-ingress-controller/add-ingresses.md

```diff
@@ -11,9 +11,8 @@ Ingresses can be added for workloads to provide load balancing, SSL termination
 1. Click **Create**.
 1. Select an existing **Namespace** from the drop-down list.
 1. Enter a **Name** for the ingress.
-1. Create ingress forwarding **Rules**. For help configuring the rules, refer to [this section.](./ingress-configuration.md#ingress-rule-configuration) If any of your ingress rules handle requests for encrypted ports, add a certificate to encrypt/decrypt communications.
+1. Create ingress forwarding **Rules**. For help configuring the rules, refer to [this section.](./ingress-configuration.md) If any of your ingress rules handle requests for encrypted ports, add a certificate to encrypt/decrypt communications.
```

Comment:
```suggestion
1. [Create](./ingress-configuration.md) ingress forwarding **Rules**. If any of your ingress rules handle requests for encrypted ports, add a certificate to encrypt/decrypt communications.
```

Should ingress be capitalized? That would be a separate PR...

Reply:
The K8s docs are a bit inconsistent about this, so the way I've approached it is capitalize if it's referring to a specific flavor (e.g. NGINX Ingress Controller) or if it's being referred to in the context of a K8s resource type (e.g. [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource) or [CustomResourceDefinition](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)). If we're fine with that approach then no change is needed here.